tests/src/testconversions.cpp

```diff
@@ -0,0 +1,115 @@
+// SPDX-FileCopyrightText: 2021-2025 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#include <config.h>
+
+#include "testhelpers.hh"
+
+#include <dune/common/test/testsuite.hh>
+
+#include <ikarus/finiteelements/mechanics/materials/strainconversions.hh>
+#include <ikarus/finiteelements/mechanics/materials/stressconversions.hh>
+#include <ikarus/utils/init.hh>
+
+using Dune::TestSuite;
+using namespace Ikarus;
+
+namespace Testing {
+constexpr auto testDeformationGradient() {
+  return Eigen::Matrix3d{
+      {1.0, 0.0, 0.5},
+      {0.0, 0.2, 0.5},
+      {0.2, 0.5, 1.0}
+  };
+}
+
+constexpr auto testStress() {
+  return Eigen::Matrix3d{
+      { 0.600872, -0.179083, 0},
+      {-0.179083,  0.859121, 0},
+      {        0,         0, 1}
+  };
+}
+
+auto testCauchyGreen() {
+  auto F = Testing::testDeformationGradient();
+  return (F.transpose() * F).eval();
+}
+} // namespace Testing
+
+auto testStressConversions() {
+  TestSuite t("Test Stress Conversions");
+  const double epsilon = 100 * std::numeric_limits<double>::epsilon();
```

Comment:
Why pre-multiplied with 100? May be it its simpler to read if it is then rather directly set to `1e-14` or similar?